WorkflowCombine/Testing/WorkerTesting.swift

```diff
@@ -21,29 +21,49 @@ import XCTest
 @testable import WorkflowCombine
 
 extension RenderTester {
-    /// Expect the given worker. It will be checked for `isEquivalent(to:)` with the requested worker.
+    /// Mock the given worker's output, and assert that the workflow's worker `isEquivalent(to:)` the `worker`.
     ///
     /// - Parameters:
-    ///   - worker: The worker to be expected
-    ///   - producingOutput: An output that will be returned when this worker is requested, if any.
+    ///   - worker: The worker that we expect was created by the workflow. Will be compared using
+    ///             `isEquivalent(to:)` to assert that the workflow's worker matches.
+    ///   - producingOutput: The output to be used instead of actually running the worker.
+    ///                      If the workflow never tries to run the worker, then this won't be used.
     ///   - key: Key to expect this `Workflow` to be rendered with.
+    @available(*, deprecated, renamed: "expectWorker", message: "Renamed")
```

Comment:
in these deprecation annotations, i think we likely want the `renamed:` parameter to fully specify the new method signature/param names so that fix-its can automatically be applied to update the relevant code. e.g. something like:

`renamed: "expectedWorker(_:mockingOutput:key:...)"`

though i didn't actually do a full integration test of this, it does seem to work locally to update existing callsites correctly when applying a fix-it.